src/platform/plugins/shared/data_view_editor/public/components/data_view_editor_flyout_content.tsx

```diff
@@ -279,6 +299,18 @@ const IndexPatternEditorFlyoutContentComponent = ({
               })}
             </EuiLink>
           )}
+          {isManaged && (
+            <EuiCallOut
+              title={i18n.translate('indexPatternEditor.managedDataViewCalloutMessage', {
+                defaultMessage:
+                  'Managed data views cannot be edited. Duplicate the data view in order to make changes.',
```

Comment:
```suggestion
                  'You can't edit managed data views. Instead, you can duplicate the data view and make changes to your newly created copy.',
```
Active voice + be unambiguous about what can be changed